examples/playwright.config.ts

```diff
@@ -17,10 +17,27 @@
 
 import { PlaywrightTestConfig } from '@playwright/test';
 
+// const RPconfig = {
+//   'token': '00000000-0000-0000-0000-000000000000',
+//   'endpoint': 'https://your.reportportal.server/api/v1',
+//   'project': 'Your project',
+//   'launch': 'Playwright test',
+//   'attributes': [
+//     {
+//       'key': 'key',
+//       'value': 'value',
+//     },
+//     {
+//       'value': 'value',
+//     },
+//   ],
+//   'description': 'Your launch description',
+// };
+
 const RPconfig = {
-  'token': '00000000-0000-0000-0000-000000000000',
-  'endpoint': 'https://your.reportportal.server/api/v1',
-  'project': 'Your project',
+  'token': '485c1ce2-6632-4ba4-bf63-ad0bf191aa25',
+  'endpoint': 'http://dev.epm-rpp.projects.epam.com:8080/api/v1',
+  'project': 'qqee',
```

Comment:
Please remove sensitive data from here and refresh your token in settings.
Also, please, doing something with comments.